app/views/devices/index.html.erb

```diff
@@ -0,0 +1,84 @@
+<%= render 'shared/partials/nav-global.html' %>
+<main class="devices">
+  <%= render 'shared/partials/header-global.html' %>
```

Comment:
@mzlock it will be global after the adjustments I make on the roasts_page branch